sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient/FormRecognizerClientOptions.cs

```diff
@@ -30,9 +30,6 @@ public FormRecognizerClientOptions(ServiceVersion version = LatestVersion)
         {
             Version = version;
             AddLoggedHeadersAndQueryParameters();
-
-            //Default Audience to Azure Public Cloud
-            Audience ??= FormRecognizerAudience.AzurePublicCloud;
```

Comment:
I am assuming that FormRecognizerClient class already handles null audience and assumes public cloud in that scenario?